src/jaxsim/math/transform.py

```diff
@@ -46,8 +46,8 @@ def from_quaternion_and_translation(
 
     @staticmethod
     def from_rotation_and_translation(
-        rotation: jtp.MatrixLike,
-        translation: jtp.VectorLike,
+        rotation: jtp.MatrixLike = jnp.eye(3),
+        translation: jtp.VectorLike = jnp.zeros(3),
```

Comment:
This may lead to a silent fail, what if we add:
```python
@classmethod 
def unitary(cls):
    return cls.from_rotation_and_translation(
        rotation=jnp.eye(3),
        translation=jnp.zeros(3),
    )
```

```suggestion
        rotation: jtp.MatrixLike,
        translation: jtp.VectorLike,
```

Reply:
The main reason for this is to pass only a rotation or only a translation, as it is already done for [`from_quaternion_and_translation`](https://github.com/ami-iit/jaxsim/blob/7340d43172d5ec528f3f7547d532ffeb9770355d/src/jaxsim/math/transform.py#L11-L17) and in the [`Adjoint`](https://github.com/ami-iit/jaxsim/blob/7340d43172d5ec528f3f7547d532ffeb9770355d/src/jaxsim/math/adjoint.py#L63-L67). Of course, if the user calls something named `from_rotation_and_translation` and does not pass a translation, to me it's implicit that the corresponding zero element should be used. I don't see room of misuse in this specific case.

Reply:
Okay, thanks for the explanation. 

Good to go for me, do you have anything else to add to the PR?